pkg/http-handler/http-handler.go

```diff
@@ -302,6 +302,12 @@ func CanI(kubeHandler kube.AuthHandler) func(w http.ResponseWriter, req *http.Re
 		_, requestCluster := getNamespaceAndCluster(req)
 
 		clientset, err := kubeHandler.AsUser(token, requestCluster)
+
+		// If the UI clusters configuration did not include the cluster on which Kubeapps is installed, two
+		// the requestCluster will be empty. We will try using the service account instead.
+		if requestCluster == "" {
+			clientset, err = kubeHandler.AsSVC(requestCluster)
+		}
```

Comment:
This sounds absolutely dangerous? Given the purpose of the `CanI` handler is to determine whether the *user*  has permission to do something, we should never be switching to the service account, I don't think?

Reply:
Mmm, that's absolutely right. I initially thought that it could be good to have fine-grained information about the rbac granted to the kubeops service account, but it may not be a great idea as pointed out.

However, I don't see that danger, once the user is authenticated, all the actions that can be performed using the service account are the same regardless of the user's privileges, aren't they?

I mean, a user with, let's say, view-only permissions, can still manually invoke those actions that eventually will fall back to the serviceaccount, no? I can't foresee any further issues apart from the privilege enumeration (albeit enough to justify removing the changes herein introduced)